cmd/inspect_policy.go

```diff
@@ -38,6 +41,8 @@ func inspectPolicyCmd() *cobra.Command {
 		outputFormat string
 	)
 
+	validFormats := []string{"json", "text", "names", "short-names"}
```

Comment:
Perhaps we need just the `name` output, akin to kubectl, other formats can be custom via templating or JSONPath...